examples/demo_io/main.bend

```diff
@@ -1,63 +1,35 @@
 test-io = 1
 
-type IO_T:
-  Done { magic, expr }
-  Call { magic, func, argm, cont }
+def unwrap(res):
+  match res:
+    case Result/Ok:
+      return res.val
+    case Result/Err:
+      return res.val
 
-def IO_T/MAGIC:
-  return (0xD0CA11, 0xFF1FF1)
+def open():
+  return call("OPEN", ("./README.md", "r"))
```

Comment:
Opening the readme is good as an example, but when running as a test it will error if we change the start of the readme.

Maybe the program reading its own source code would be better